test/boost/server_stop_without_listening_test.cc

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
```

Comment:
Please name the file generic_server_test.cc, so if we ever have related tests, they'll have a home.

Reply:
I was about 101% sure that I would never get the name right at first try, so it was a "consciously tentative" name ;)

Rename done in v6.